editor/editor_node.cpp

```diff
@@ -1244,7 +1244,8 @@ void EditorNode::save_resource_in_path(const Ref<Resource> &p_resource, const St
 }
 
 void EditorNode::save_resource(const Ref<Resource> &p_resource) {
-	if (p_resource->get_path().is_resource_file()) {
+	// If the resource has been imported, ask the user to use a different path in order to save it.
+	if (p_resource->get_path().is_resource_file() && !FileAccess::exists(p_resource->get_path() + ".import")) {
 		save_resource_in_path(p_resource, p_resource->get_path());
```

Comment:
Nitpick but this could use a `String path = p_resource->get_path();` since it ends up accessed twice in the `if` condition and once more if true.